### PR TITLE
add card

### PR DESCRIPTION
**Pull request by manch0ffline**
- [DEMO LINK](https://manch0ffline.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://manch0ffline.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -1,3 +1,133 @@
...
 body {
   margin: 0;
+  font-family: Manrope, sans-serif;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: $color-secondary;
+}
+
+.card {
+  box-sizing: border-box;
```
Comment (OleziO) [inline src/styles/index.scss:18]: Move to the * selector and remove it from all other places.

**Review by OleziO — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,53 @@
...
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="card__img"
+      >
+      <div class="card__info">
+        <h1 class="card__title">
```
Comment (anastasiiavorobiova) [inline src/index.html:21]:
```suggestion
        <h2 class="card__title">
```

Comment (anastasiiavorobiova) [reply src/index.html:21]: It's expected to have a lot of cards on the page

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,53 @@
...
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <div class="card__product-code">
```
Comment (anastasiiavorobiova) [inline src/index.html:24]:
```suggestion
        <p class="card__product-code">
```
Consider using semantic tags for the text

File: src/index.html [hunk 4]
```diff
@@ -5,9 +5,53 @@
...
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
```
Comment (anastasiiavorobiova) [inline src/index.html:14]:
```suggestion
    <article class="card" data-qa="card">
```
Consider leveraging semantic tags

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,136 @@
...
+    background: url(../images/star.svg) no-repeat center;
+    margin-right: 4px;
+  }
+
+  &__stars-star:not(:last-child) {
+    margin-right: 4px;
+
+  }
+
+  &__stars--4 :nth-child(-n+4){
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:121]: Consider keeping modifiers functionality from the previous task

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Review by l4st1m0za — APPROVED** [review]
Preatty good job, but consider the comments for the further tasks

**Pull request closed without merging**